core/chains/evm/txmgr/eth_broadcaster.go

```diff
@@ -556,6 +556,16 @@ func (eb *EthBroadcaster) handleInProgressEthTx(ctx context.Context, etx EthTx,
 		return errors.Wrap(sendError, "this error type only handled for L2s"), false
 	}
 
+	if sendError.IsNonceTooHighError() {
```

Comment:
Can you please also add a unit-test for this case, as part of the test "TestEthBroadcaster_ProcessUnstartedEthTxs_Errors"?

I see there are already other cases which have tests in that function.